related.md

```diff
@@ -11,14 +11,19 @@ This also contains the official specification of the [OME Zarr format](https://g
 ## [idr-zarr-tools](https://github.com/IDR/idr-zarr-tools)
 A full workflow demonstrating the conversion of IDR images to OME Zarr images on S3.
 
+## [zarr.md](https://github.com/IDR/SubmissionWorkflow/blob/master/zarr.md)
```

Comment:
This is a private repo

Reply:
Doh. Though hopefully not for long. I'll re-open when&if.